package.json

```diff
@@ -16,8 +16,7 @@
     "build": "yarn workspaces foreach -vpt run build",
     "test": "yarn workspaces foreach -vpt run test",
     "lint": "yarn workspaces foreach -vpt run lint",
-    "lint:style:fix": "yarn workspaces foreach -vp run lint:style:fix",
-    "demo": "concurrently  --raw --kill-others \"yarn run start:snap\" \"yarn run start:example\""
+    "lint:style:fix": "yarn workspaces foreach -vp run lint:style:fix"
```

Comment:
we need to specify node version >16.10 if we are mentioning core pack in the documentation